README.md

```diff
@@ -72,12 +82,14 @@ py_binary(
 )
```

Comment:
Replace the dots with: `srcs = ["main.py"]`

Reply:
Done.